code/controllers/subsystem/shuttles/emergency.dm

```diff
@@ -148,3 +146,27 @@
 		if(.)	return .
 		return ..()
 */
+
+/obj/machinery/computer/shuttle/pod
```

Comment:
Because the rest of the emergency shuttle code is there, and all the special shuttle computers (such as the gulag) are in their own files anyway